js-exercises/sum-primes/sumPrimes.js

```diff
@@ -0,0 +1,24 @@
+function sumPrimes(num) {
+  let prime = [];
```

Comment:
prime sounds like a singular number, whereas this is an array. `primes`, `primeNumbers` etc. would be better names here.